packages/gdl-frontend/components/GlobalMenu/index.js

```diff
@@ -55,8 +55,8 @@ class GlobalMenu extends React.Component<Props, State> {
     const { onClose } = this.props;
 
     return (
-      <Drawer open={this.props.isOpen} onClose={onClose}>
-        <List component="nav">
+      <Drawer open={this.props.isOpen} onClose={onClose} label="Open menu">
```

Comment:
`Open menu` should be the label for the button that opens the drawer, not the drawer itself